### DISCUSSION TITLE
Update Footer/Colophon Area

### DISCUSSION THREAD

**Discussion by bacoords** [holger1411/understrap#1363] [Ideas]
I think it's time we make the bottom footer text editable for the average WP user. I'm talking specifically about the `understrap_site_info();` function in `footer.php`. I know it's editable via child theme, but I really think a default WP user should be able to modify that text.

I think there's a few options here: 
- A customizer setting for the footer. We add a textarea field in the customizer. If used, it overwrites the return of that function. If not, everything stays the same.
- Register a footer nav menu. If used, it replaces `understrap_site_info()` with a menu. Perfect for links to terms, privacy policy, etc. If it's no menu is assigned, it stays the same.

Other ideas? Should we just leave it alone?

**Comment by danemorgan** [comment]
We have a [related discussion #1332 ](https://github.com/understrap/understrap/discussions/1332) requesting multiple menu locations. I'm not sure I go in for all of them, but I think a colophon would not be a bad idea at all.

I wonder whether there is a use case for both menu and textual options available at that location.

**Reply by bacoords** [reply]
Yeah with bootstrap grid you can easily do an optional footer menu as another column that pushes the copyright text to the left if the menu is in use...

**Reply by 0dp** [reply]
Colophon should be editable in customizer imo

**Comment by bacoords** [comment]
I guess the third option I should've added is 'start working on a full-site editing branch' which would allow users to update the header/footer that way.. but.. yikes..

**Reply by danemorgan** [reply]
I sort of think that's a mid range goal.
